### ISSUE TITLE
Gem failed to install for macOS 13 Ventura, Apple M2 chip with Ruby 3.1.3.

### ISSUE THREAD

**Issue by bweathers3** [ruby-gnome/ruby-gnome#1529]
Using: macOS 13 Ventura, Apple M2 chip, for RoR the following gems were not able to install with Ruby 3.1.3.

I was unable to bundle the following gems: rsvg2, glib2, gobject-introspection

I believe the error stated - failed to build the native extensions.

However with Ruby 2.7.7 the gems did install on the system.

**Comment by kou** [comment]
Could you show the full error message and log?

**Comment by bweathers3** [comment]
Sutou,

So the laptop we were setting up was my son's which I don't have access to
currently.

I had cloned one of my current projects to his machine for him to be able
to review while he is taking a class. There are no problems running the
project on both my machines which are Intel Mac's.

Unfortunately, the only way to get the project working on his machine was
to downgrade ruby for the whole project to 2.7.7.

I did not keep the logs and would not have thought about submitting an
issue except after but it was recommended by the  service called "Ruby on
Macs". I used them to set up my son's machine thinking I had made an
installation mistake. After resetting his machine and using the service to
reset up my son's laptop we still had the same issues trying to bundle the
gems. From there we downgraded the project.

So in the end I did not take time to save or copy the logs, my thinking was
that I had made the mistake. I hope that makes sense.

If you have not had other reports maybe just leave to post up a little
while?????

Thanks,

Bailey
